services/efs-guardian/tests/unit/conftest.py

```diff
@@ -140,23 +129,37 @@ async def client(app: FastAPI) -> AsyncIterator[httpx.AsyncClient]:
         yield client
 
 
+#
+# Redis
+#
+
+
 @pytest.fixture
-async def rpc_client(
-    rabbit_service: RabbitSettings,
-    app: FastAPI,
-    rabbitmq_rpc_client: Callable[[str], Awaitable[RabbitMQRPCClient]],
-) -> RabbitMQRPCClient:
-    return await rabbitmq_rpc_client("client")
+def disable_redis_and_background_tasks_setup(mocker: MockerFixture) -> Callable:
+    def _():
+        # The following services are affected if redis is not in place
+        mocker.patch("simcore_service_efs_guardian.core.application.setup_redis")
+        mocker.patch(
+            "simcore_service_efs_guardian.core.application.setup_background_tasks"
+        )
+
+    return _
 
 
 @pytest.fixture
-async def mocked_redis_server(mocker: MockerFixture) -> None:
-    mock_redis = FakeRedis()
-    mocker.patch("redis.asyncio.from_url", return_value=mock_redis)
+def with_disabled_redis_and_background_tasks(
+    disable_redis_and_background_tasks_setup: Callable,
+):
+    disable_redis_and_background_tasks_setup()
+
+
+#
+# Others
+#
 
 
 @pytest.fixture
-async def cleanup(app: FastAPI):
+async def efs_cleanup(app: FastAPI):
```

Comment:
TIP: `Path` incorporates most of the functionality previously in `os` and `os.path`. Use the former e.g. herre you yse `dir_path.exists()` but you can also use `dir_path.walk()`


Then automatically all the outputs are also `Path`! and you do not need to convert them as you do here

SEE https://docs.python.org/3/library/pathlib.html#corresponding-tools

Reply:
We need to wait for Python 3.12 ;)